src/com/yetanalytics/lrs_admin_ui/views/status.cljs

```diff
@@ -91,17 +91,16 @@
           :orientation "bottom"}]]
        loading?
        [:div]
-
        :else
-       [:div.no-data "No Statement Data"])]))
+       [:div.no-data @(subscribe [:lang/get :monitor.no-data])])]))
 
 (defn timeline-select-unit
   []
   (let [current-unit @(subscribe [:status.params/timeline-unit])]
     [:div.vis-timeline-controls-select-unit
      [:label
       {:for "timeline-select-unit"}
-      "Time Unit"]
+      @(subscribe [:lang/get :monitor.timeline.unit])]
      (into [:select
             {:id "timeline-select-unit"
              :value current-unit
```

Comment:
This is all well and good, but could the time units themselves also be translated?

(Also why doesn't GitHub allow us to comment on unchanged lines? 😠 )